inline_scan.sh

```diff
@@ -133,10 +131,10 @@ get_and_validate_analyzer_options() {
             C  ) clean_flag=true;;
             V  ) V_flag=true;;
```

Comment:
Out of the scope of this PR, but I also think that the v_flag and setting VERBOSE=true has no effect. If we want verbose output we should add "--debug" to the anchore-manager command.

Reply:
see above.